packages/suite/src/components/suite/PrerequisitesGuide/PrerequisitesGuide.tsx

```diff
@@ -1,114 +1,122 @@
-                case 'device-unacquired':
-                case 'device-busy':
-                    return <DeviceAcquire />;
-                case 'device-thp-locked':
-                    return <DeviceTrezorHostProtocolPair />;
-                case 'device-used-elsewhere':
-                    return <DeviceUsedElsewhere />;
-                case 'device-unreadable':
-                    return <DeviceUnreadable />;
-                case 'device-unknown':
-                    return <DeviceUnknown />;
-                case 'device-seedless':
-                    return <DeviceSeedless />;
-                case 'device-recovery-mode':
-                    return <DeviceRecoveryMode />;
-                case 'device-initialize':
-                    return <DeviceInitialize />;
-                case 'device-bootloader':
-                    return <DeviceBootloader />;
-                case 'firmware-missing':
-                    return <DeviceNoFirmware />;
-                case 'firmware-required':
-                    return <DeviceUpdateRequired />;
-                case 'multi-share-backup-in-progress':
-                    return <MultiShareBackupInProgress />;
-                default:
-                    return <></>;
-            }
-        },
-        [prerequisite],
+    const [prerequisite, setPrerequisite] = useState<ReturnType<typeof selectPrerequisite>>(
+        useSelector(selectPrerequisite),
     );
+    const [deviceStatus, setDeviceStatus] = useState<ReturnType<typeof getStatus> | null>(
+        (device && getStatus(device)) ?? null,
+    );
+    const selectedDevice = useSelector(selectSelectedDevice);
+
+    const showWarning =
+        !!(device && deviceStatus && deviceNeedsAttention(deviceStatus)) ||
+        prerequisite === 'no-transport';
+    const showWarningIcon = shouldDisplayInitialWarningIcon(deviceStatus);
 
-    const deviceStatus = (device && getStatus(device)) ?? null;
+    const texts = getMessageId({
+        connected: !!device,
+        showWarning: showWarningIcon ?? showWarning,
+        deviceStatus,
+        prerequisite,
+    });
 
     return (
-        <Column alignItems="center" gap={spacings.xxxl} margin={{ vertical: 40 }}>
-            <ConnectDevicePrompt
-                connected={!!device}
-                deviceStatus={deviceStatus}
-                showWarning={
-                    !!(device && deviceStatus && deviceNeedsAttention(deviceStatus)) ||
-                    prerequisite === 'no-transport'
-                }
-                showWarningIcon={shouldDisplayInitialWarningIcon(deviceStatus)}
+        <>
+            <TopAnimation>
+                <Column alignItems="center">
+                    {showDeviceImage && <DeviceImage selectedDevice={selectedDevice} />}
+                    <Heading>
+                        <Translation id={texts.heading} />
+                    </Heading>
+                    {texts.description && <Description>{texts.description}</Description>}
```

Comment:
`{text.description}` must be inside `<Translation>`

<img width="608" height="599" alt="Image" src="https://github.com/user-attachments/assets/0b19ba27-53d7-4561-a57d-5496b8e415ad" />